public/modules/custom/grants_oma_asiointi/templates/grants-oma-asiointi-block.html.twig

```diff
@@ -9,7 +9,7 @@
 </div>
 <div class="content">
   <div class="messages">
-    <h4>{{ "New messages"|t }} <span class="new-messages-length {% if messages %}has-new-messages {% endif %}">{{ messages|length }}</span> </h4>
+    <h4>{{ "New on your applications"|t }} <span class="new-messages-length {% if messages %}has-new-messages {% endif %}">{{ messages|length }}</span> </h4>
```

Comment:
Oon melko varma, että tähän on joku tapa tulostaa pluralit tossa t() metodissa. 

Jos tää pelaa ni mä nyt sit kuitenki mergeen tän ni saadaan eteenpäin.